src/wp-includes/blocks.php

```diff
@@ -760,6 +760,29 @@ function get_hooked_blocks( $name ) {
 	return $hooked_blocks;
 }
 
+function insert_hooked_blocks( $block, $parent, $block_index, $chunk_index ) {
+	$hooked_blocks = get_hooked_blocks( $block['blockName'] );
+	foreach ( $hooked_blocks as $hooked_block_type => $relative_position ) {
+		$hooked_block = array(
+			'blockName'    => $hooked_block_type,
+			'attrs'        => array(),
+			'innerHTML'    => '',
+			'innerContent' => array(),
+			'innerBlocks'  => array(),
+		);
+		if ( 'before' === $relative_position ) {
+			insert_inner_block( $parent, $block_index, $chunk_index, $hooked_block );
+		} elseif ( 'after' === $relative_position ) {
+			insert_inner_block( $parent, $block_index + 1, $chunk_index, $hooked_block );
```

Comment:
Add checks to make sure `$parent` (and `$block_index` and `$chunk_index`) exist -- 'cause they don't [if `$block` is at the top level of the tree](https://github.com/WordPress/wordpress-develop/pull/5246/files#diff-8c99af92e4ec0fdb307ddd9b42be1e1ef1efe4a9f31287c23f346244dddd1ce9R954-R955) 😬